### ISSUE TITLE
[Paper] Broken Neural Scaling Laws

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed-dev#145]
{
  "arxivId": "2210.14891",
  "url": "https://arxiv.org/abs/2210.14891",
  "title": "Broken Neural Scaling Laws",
  "authors": "Ethan Caballero, Kshitij Gupta, Irina Rish, David Krueger",
  "abstract": "We present a smoothly broken power law functional form (that we refer to as a\nBroken Neural Scaling Law (BNSL)) that accurately models &amp; extrapolates the\nscaling behaviors of deep neural networks (i.e. how the evaluation metric of\ninterest varies as amount of compute used for training (or inference), number\nof model parameters, training dataset size, model input size, number of\ntraining steps, or upstream performance varies) for various architectures &amp; for\neach of various tasks within a large &amp; diverse set of upstream &amp; downstream\ntasks, in zero-shot, prompted, &amp; finetuned settings. This set includes\nlarge-scale vision, language, audio, video, diffusion, generative modeling,\nmultimodal learning, contrastive learning, AI alignment, AI capabilities,\nrobotics, out-of-distribution (OOD) generalization, continual learning,\ntransfer learning, uncertainty estimation / calibration, OOD detection,\nadversarial robustness, distillation, sparsity, retrieval, quantization,\npruning, fairness, molecules, computer programming/coding, math word problems,\n\"emergent phase transitions\", arithmetic, supervised learning,\nunsupervised/self-supervised learning, &amp; reinforcement learning (single agent &amp;\nmulti-agent). When compared to other functional forms for neural scaling, this\nfunctional form yields extrapolations of scaling behavior that are considerably\nmore accurate on this set. Moreover, this functional form accurately models &amp;\nextrapolates scaling behavior that other functional forms are incapable of\nexpressing such as the nonmonotonic transitions present in the scaling behavior\nof phenomena such as double descent &amp; the delayed, sharp inflection points\npresent in the scaling behavior of tasks such as arithmetic. Lastly, we use\nthis functional form to glean insights about the limit of the predictability of\nscaling behavior. Code is available at\nhttps://github.com/ethancaballero/broken_neural_scaling_laws",
  "timestamp": "2024-12-21T05:51:08.748Z",
  "rating": "downvote"
}

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.
